src/main/java/xyz/thepathfinder/android/Model.java

```diff
@@ -224,24 +244,13 @@ private boolean updateType(String reason, JsonObject json) {
     @Override
     @SuppressWarnings("unchecked")
     protected boolean notifyUpdate(String reason, JsonObject json) {
-        if (!json.has("model") || !json.get("model").getAsString().equals(this.getModel())) {
+/*        if (!json.has("model") || !json.get("model").getAsString().equals(this.getModel())) {
```

Comment:
All the commented out code is for when Dan changes the server to use one field for the path.